client/components/AllProducts.js

```diff
@@ -46,11 +46,13 @@ class AllProducts extends Component {
         <NavLink to={`/products/${product.id}`} key={product.id}>
           <div className="col s12 m6 l3">
             <div className="card small">
-              <div className="card-image">
+              <div className="cardimage">
```

Comment:
is it card-image or cardimage? I'm seeing card-image on the materialize docs: https://materializecss.com/cards.html